### DISCUSSION TITLE
Brightness control failed on Picasso platform

### DISCUSSION THREAD

**Discussion by ovoME** [ChefKissInc/NootedRed#274] [Q&A] [answered]
3550H, Vega 8, macOS 12
I could turn up/down when I start the OS, but when I wake up macOS from sleeping I could not control the brightness
The brightness stuck on about 80%
Did anyone have this bug?

**Comment by VisualEhrmanntraut** [comment, marked as answer]
#236

**Comment by ovoME** [comment]
I thought an idea：maybe we can design a command for users to add boot-args to disable Catalina brightness control code, or delete Catalina brightness control code, because Catalina is outdated.

(I'm sure it's caused by the Catalina brightness control code, because when you released the kext with the Catalina brightness control code, this problem appeared)
